docs/core/tutorials/with-visual-studio-code.md

```diff
@@ -68,7 +68,7 @@ You can also watch a short video tutorial for further setup help on [Windows](ht
 
     ![Selecting .NET Core](media/with-visual-studio-code/selectcore.png)
 
-5. Add a breakpoint to your project by clicking on the **editor margin**, which is the space on the left of the line numbers in the editor, next to line 9.
+5. Add a breakpoint to your project by clicking on the **editor margin**, which is the space on the left of the line numbers in the editor, next to line 9, or put mouse cursor on line 9 and press <kbd>F9</kbd>.
```

Comment:
Where the breakpoint is placed when you press <kbd>F9</kbd> is not controlled by the mouse cursor, but by the text cursor. So I think it's confusing to talk about the mouse cursor here.